### PR TITLE
fix: Remove html tags from speaker's bio.

### LINKED ISSUE #1627: Html tags in speaker details.
**Describe the bug**

Html tags in speaker details.

**Screenshots**

image

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #1627 

Changes: Removed html tags.

Screenshots for the change:
image

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/speakers/SpeakerViewHolder.kt [hunk 1]
```diff
@@ -10,13 +10,14 @@ import kotlinx.android.synthetic.main.item_speaker.view.speakerOrgTv
...
 import org.fossasia.openevent.general.utils.Utils
+import org.fossasia.openevent.general.utils.stripHtml
 
 class SpeakerViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
 
     fun bind(speaker: Speaker) {
         itemView.speakerNameTv.text = speaker.name
         itemView.speakerOrgTv.text = speaker.organisation
-        itemView.shortBioTv.text = speaker.shortBiography
+        itemView.shortBioTv.text = speaker.shortBiography?.stripHtml()
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/speakers/SpeakerViewHolder.kt]: Do this automatically in the class itself

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/speakers/SpeakerViewHolder.kt]: Okay

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/speakers/SpeakerViewHolder.kt]: done

File: app/src/main/java/org/fossasia/openevent/general/speakers/Speaker.kt [hunk 2]
```diff
@@ -18,9 +19,9 @@ data class Speaker(
     val name: String?,
     val email: String?,
     val photoUrl: String?,
-    val shortBiography: String?,
-    val longBiography: String?,
-    val speakingExperience: String?,
+    var shortBiography: String?,
+    var longBiography: String?,
+    var speakingExperience: String?,
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/speakers/Speaker.kt]: See how it is done for event description

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/speakers/Speaker.kt]: val description = event.description.nullToEmpty().stripHtml()
        if (!description.isNullOrEmpty()) {
            setTextField(rootView.eventDescription, description)

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/speakers/Speaker.kt]: It's done like this for event description. Should I do it like this?

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/speakers/Speaker.kt]: Yes

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/speakers/Speaker.kt]: done

**Pull request merged**